test/sharness/t0100-name.sh

```diff
@@ -32,6 +32,16 @@ test_expect_success "resolve output looks good" '
   test_cmp expected2 output
 '
 
+# also test resolving the "self" key
+
+test_expect_success "'ipfs name resolve self' succeeds" '
+  ipfs name resolve self >self_output
+'
+
+test_expect_success "self resolve output looks good" '
+  test_cmp expected2 self_output
+'
+
```

Comment:
Could you add tests with generated keys as well? Not only the `self` key?

Reply:
@Kubuxu,  It's been 6 months, Can I take over his job?

Reply:
@overbool Yes, go for it!